src/groovy/au/org/emii/portal/wms/GeoserverServer.groovy

```diff
@@ -10,6 +10,12 @@ package au.org.emii.portal.wms
 import au.org.emii.portal.proxying.ExternalRequest
 
 class GeoserverServer extends WmsServer {
+    def dynamicFilters
+
+    GeoserverServer(dynamicFilters) {
+        this.dynamicFilters = dynamicFilters
```

Comment:
Yeah we'll configure GeoServer filter plugins per server and get rid of this here.